server/form-pages/apply/risks-and-needs/risk-of-serious-harm/riskManagementArrangements.ts

```diff
@@ -28,7 +28,9 @@ export type RiskManagementArrangementsBody = {
 export default class RiskManagementArrangements implements TaskListPage {
   documentTitle = 'Risk management arrangements for the person'
```

Comment:
do we want 'for the person' on all our document titles? I think atm we're not completely consistent (but I would prefer to remove it because I think it adds nothing)